SPECS/kubevirt/kubevirt.spec

```
@@ -0,0 +1,215 @@
+#
+# spec file for package kubevirt
+#
+# Copyright (c) 2022 SUSE LLC
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+
+%global debug_package %{nil}
+Summary:        Container native virtualization
+Name:           kubevirt
+Version:        0.51.0
+Release:        1%{?dist}
+License:        Apache-2.0
```

We try to use [Fedora's "short names"](https://fedoraproject.org/wiki/Licensing:Main#Software_License_List) when referencing licenses, to keep things consistent.

```suggestion
License:        ASL 2.0
```